spec/factories.rb

```diff
@@ -29,18 +33,33 @@
   end
 
   factory :move do
-    sequence(:name) { |n| "Sybil #{n}th" }
+    sequence(:name) { |n| "#{Faker::App.name} #{rand(1000)}th" }
     user
     description "My favorite move"
   end
 
   factory :video do
-    sequence(:name) { |n| "Classic #{n}th" }
+    sequence(:name) { |n| "#{Faker::App.name} #{rand(1000)}" }
```

Comment:
I am happy to use `n` but I used rand(1000) just so it lessens that instance. The problem with using `n` is you can't run the seeds again, as it starts from 0 (or 1?) again and Faker's name generator usually makes a hit. In my experience this is far more common than `rand` + Faker having the same name.